curriculum/challenges/english/06-quality-assurance/advanced-node-and-express/create-new-middleware.english.md

```diff
@@ -8,9 +8,9 @@ forumTopicId: 301551
 ## Description
 <section id='description'>
 
-As in, any user can just go to /profile whether they authenticated or not by typing in the url. We want to prevent this by checking if the user is authenticated first before rendering the profile page. This is the perfect example of when to create a middleware.
+As is, any user can just go to <em>/profile</em> whether they have authenticated or not, by typing in the url. We want to prevent this, by checking if the user is authenticated first before rendering the profile page. This is the perfect example of when to create a middleware.
```

Comment:
```suggestion
As is, any user can just go to <code>/profile</code> whether they have authenticated or not, by typing in the url. We want to prevent this, by checking if the user is authenticated first before rendering the profile page. This is the perfect example of when to create a middleware.
```